libraries/RainMaker/examples/RMakerCustom/RMakerCustom.ino

```diff
@@ -26,24 +26,67 @@ static Device my_device("Dimmer", "custom.device.dimmer", &gpio_dimmer);
 
 void sysProvEvent(arduino_event_t *sys_event)
 {
-    switch (sys_event->event_id) {
-        case ARDUINO_EVENT_PROV_START:
-#if CONFIG_IDF_TARGET_ESP32S2
-            Serial.printf("\nProvisioning Started with name \"%s\" and PoP \"%s\" on SoftAP\n", service_name, pop);
-            printQR(service_name, pop, "softap");
-#else
-            Serial.printf("\nProvisioning Started with name \"%s\" and PoP \"%s\" on BLE\n", service_name, pop);
-            printQR(service_name, pop, "ble");
-#endif
-            break;
-        case ARDUINO_EVENT_PROV_INIT:
-            wifi_prov_mgr_disable_auto_stop(10000);
```

Comment:
This code was added to handle provisioning failures when the log level is set to "Debug". This shouldn't be removed.